src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const headers = Array.from(document.querySelector('tr').children);
+
+const parentHeader = headers[0].parentElement;
+const clonedHeader = headers[1].cloneNode(true);
+
+parentHeader.insertBefore(clonedHeader, headers[4]);
```

Comment:
Ensure that `headers[4]` exists before attempting to insert `clonedHeader` before it. If `headers` has fewer than 5 elements, this will cause an error.